cmd/opfgd/start.go

```diff
@@ -86,14 +85,6 @@ func runStartCmd(ctx client.Context, cmd *cobra.Command, args []string) error {
 		}
 	}()
 
-	// Create grpc client
```

Comment:
is this change to break some circular dependency?

Reply:
no, this change only removes the unnecessary grpc client.